#!/usr/bin/env python

PACKAGE = 'neatness_estimator'

from dynamic_reconfigure.parameter_generator_catkin import *;

gen = ParameterGenerator ()
gen.add("max_size", int_t, 0, "the max number of the points of each cluster", 25000, 0, 2000000)
gen.add("min_size", int_t, 0, "the min number of the points of each cluster", 20, 0, 1000)
gen.add("tolerance", double_t, 0, "margin length of each clustering", 0.02, 0.0, 1.0)

exit (gen.generate (PACKAGE, "neatness_estimator", "MultiEuclideanClustering"))
